.github/workflows/check.yml

```diff
@@ -71,9 +82,11 @@ jobs:
       - name: cargo install cargo-hack
         uses: taiki-e/install-action@cargo-hack
       # intentionally no target specifier; see https://github.com/jonhoo/rust-ci-conf/pull/4
+      # --feature-powerset runs for every combination of features
       - name: cargo hack
         run: cargo hack --feature-powerset check
   msrv:
+    # determine the minimal rust version that is supported by this crate
```

Comment:
Ah, this isn't quite right: this doesn't _determine_ the MSRV, it checks whether the configured MSRV can indeed still build the current crate. So it'll warn you if you accidentally bump the MSRV, but it will not _find_ the MSRV in such a case.